spec/models/photo_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe Photo, type: :model do
+
+  it { is_expected.to belong_to :user }
+  it { is_expected.to have_many :comments }
+  it { is_expected.to have_many :likes }
+
+  describe 'build_with_user' do
+
+    let(:user) { create(:user) }
+
+    let(:photo) { Photo.create title: "test",
+                  image:  Rack::Test::UploadedFile.new(Rails.root + 'spec/assets_specs/photos/test_photo1.jpg', 'image/jpg')}
```

Comment:
Line is too long. [125/80]<br>Align the elements of a hash literal if they span more than one line.<br>Expression at 14, 125 should be on its own line.<br>Unnecessary spacing detected.